modules/microshift-low-latency-config-yaml.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-low-latency-config-yaml_{context}"]
+= Configuration kubelet parameters and values in {microshift-short}
+
+The first step in enabling low latency to a {microshift-short} cluster is to add configurations to the {microshift-short} `config.yaml` file.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have root access to the cluster.
+* You made a copy of the provided `config.yaml.default` file in the `/etc/microshift/` directory, renaming it `config.yaml`.
+
+.Procedure
+
+. Add the kubelet configuration to the {microshift-short} `config.yaml` file
++
+.Example passthrough `kubelet` configuration
+[source,yaml]
+----
+# ...
+kubelet: <1>
+  cpuManagerPolicy: static # <2>
+  cpuManagerPolicyOptions:
+    full-pcpus-only: "true" # <3>
+  cpuManagerReconcilePeriod: 5s
+  memoryManagerPolicy: static # <4>
+  topologyManagerPolicy: single-numa-node
+  reservedSystemCPUs: 0-1 # <5>
+  reservedMemory:
+  - limits:
+      memory: 1100Mi # <6>
+    numaNode: 0
+  kubeReserved:
+    memory: 500Mi
+  systemReserved:
+    memory: 500Mi
+  evictionHard: # <7>
+    imagefs.available: "15%" # <8>
+    memory.available: "100Mi" # <9>
+    nodefs.available: "10%" # <10>
+    nodefs.inodesFree: "5%" # <11>
+  evictionPressureTransitionPeriod: 0s
+# ...
+----
+<1> If you change the CPU or memory managers in the kubelet configuration, you must remove files that cache the previous configuration. Restart the host to remove them automatically, or manually remove the `- /var/lib/kubelet/cpu_manager_state` and `- /var/lib/kubelet/memory_manager_state` files.
+<2> The name of the policy to use. Valid values are `none` and `static`. Requires the `CPUManager` feature gate to be enabled. Default value is `none`.
+<3> A set of `key=value` for setting extra options that fine tune the behavior of the `CPUManager` policies.  Default: null. Requires both the `CPUManager` and `CPUManagerPolicyOptions` feature gates to be enabled.
+<4> The name of the policy used by Memory Manager. Default value is `none`. Requires the `MemoryManager` feature gate to be enabled.
+<5> Required. This parameter is essential to dividing the management and application workloads. Use this parameter to define a static CPU set for the host-level system and Kubernetes daemons, plus interrupts and timers. Then the rest of the CPUs on the system can be used exclusively for workloads. Use your TuneD profile to move the system and Kubernetes daemons, interrupts, and timers to the explicit CPU set defined by this option.
```

Comment:
Do we want to mention that reservedSystemCPUs should be the inverse of the isolated CPUs? As both combined need to account for all CPUs on the system.

Reply:
I think it's mentioned in other place that they need to be complementary. However, this is not so clear if you're offlining CPUs.
@ShaunaDiaz I just checked the parent tuned profile (cpu-partitioning) and it specifically checks if `isolated_cores` have `offline` CPUs in it.
However, cpu-partitioning is activated before our profile, it doesn't seem to break anything (first a core is isolated, then offlined).
But I cannot vouch that it'll not break something somewhere...